sql/moz-fx-data-shared-prod/telemetry/clients_last_seen_v2/view.sql

```diff
@@ -1,26 +1,140 @@
+  -- consecutive days each week for six weeks; the current day and previous day are accepted.
+  BIT_COUNT(days_seen_bits & 0x0FFFFFFE) >= 14 AS is_regular_user_v3,
+  days_seen_bits & 0x0FFFFFFE = 0 AS is_new_or_resurrected_v3,
+  BIT_COUNT(days_seen_bits & 0x0FFFFFFE) >= 14
+  AND (
+    (
+      BIT_COUNT(
+        days_seen_bits & 0x0FFFFFFE & (
+          0x183060C183 >> (8 - EXTRACT(DAYOFWEEK FROM submission_date))
+        )
+      ) <= 1
+    )
+    OR (
+      BIT_COUNT(
+        days_seen_bits & 0x0FFFFFFE & (
+          0x183060C183 >> (8 - EXTRACT(DAYOFWEEK FROM submission_date) - 1)
+        )
+      ) <= 1
+    )
+    OR (
+      BIT_COUNT(
+        days_seen_bits & 0x0FFFFFFE & (
+          0x183060C183 >> (8 - EXTRACT(DAYOFWEEK FROM submission_date) + 1)
+        )
+      ) <= 1
+    )
+  ) AS is_weekday_regular_v1,
+  BIT_COUNT(days_seen_bits & 0x0FFFFFFE) >= 14
+  AND NOT (
+    (
+      BIT_COUNT(
+        days_seen_bits & 0x0FFFFFFE & (
+          0x183060C183 >> (8 - EXTRACT(DAYOFWEEK FROM submission_date))
+        )
+      ) <= 1
+    )
+    OR (
+      BIT_COUNT(
+        days_seen_bits & 0x0FFFFFFE & (
+          0x183060C183 >> (8 - EXTRACT(DAYOFWEEK FROM submission_date) - 1)
+        )
+      ) <= 1
+    )
+    OR (
+      BIT_COUNT(
+        days_seen_bits & 0x0FFFFFFE & (
+          0x183060C183 >> (8 - EXTRACT(DAYOFWEEK FROM submission_date) + 1)
+        )
+      ) <= 1
+    )
+  ) AS is_allweek_regular_v1,
+  BIT_COUNT(days_visited_1_uri_bits) >= 21 AS is_core_active_v1,
   CASE
-    WHEN cls.isp_name = 'BrowserStack'
-      THEN CONCAT(cls.app_name, ' ', cls.isp_name)
-    WHEN distribution_id = 'MozillaOnline'
-      THEN CONCAT(cls.app_name, ' ', cls.distribution_id)
-    ELSE cls.app_name
-  END AS app_name,
-  IFNULL(mozfun.bits28.days_since_seen(cls.days_active_bits) = 0, FALSE) AS is_dau,
-  IFNULL(mozfun.bits28.days_since_seen(cls.days_active_bits) < 7, FALSE) AS is_wau,
-  IFNULL(mozfun.bits28.days_since_seen(cls.days_active_bits) < 28, FALSE) AS is_mau,
-  IFNULL(mozfun.bits28.days_since_seen(cls.days_seen_bits) = 0, FALSE) AS is_daily_user,
-  IFNULL(mozfun.bits28.days_since_seen(cls.days_active_bits) < 7, FALSE) AS is_weekly_user,
-  IFNULL(mozfun.bits28.days_since_seen(cls.days_seen_bits) < 28, FALSE) AS is_monthly_user,
-  IF(
-    LOWER(IFNULL(cls.isp_name, '')) <> "browserstack"
-    AND LOWER(IFNULL(cls.distribution_id, '')) <> "mozillaonline",
-    TRUE,
-    FALSE
-  ) AS is_desktop
+    WHEN BIT_COUNT(days_visited_1_uri_bits)
+      BETWEEN 1
+      AND 6
+      THEN 'infrequent_user'
+    WHEN BIT_COUNT(days_visited_1_uri_bits)
+      BETWEEN 7
+      AND 13
+      THEN 'casual_user'
+    WHEN BIT_COUNT(days_visited_1_uri_bits)
+      BETWEEN 14
+      AND 20
+      THEN 'regular_user'
+    WHEN BIT_COUNT(days_visited_1_uri_bits) >= 21
+      THEN 'core_user'
+    ELSE 'other'
+  END AS activity_segments_v1,
```

Comment:
Not sure what v1 means here. It makes sense to me to just change it to v2 or remove the suffix
```suggestion
  END AS activity_segments,
```

Reply:
I don't want to change the column names for downstream implications, would prefer to leave them as is since a lot of stuff already uses this view